scripts/deploy.ts

```diff
@@ -76,17 +76,19 @@ async function main() {
   output.walletFactory = walletFactory.address;
   console.log('WalletFactory deployed at ' + walletFactory.address);
 
-  const Forwarder = await ethers.getContractFactory('Forwarder');
+  const Forwarder = await ethers.getContractFactory('ForwarderV4');
```

Comment:
Is this a deploy function which common for everything?, If we have to deploy a forwarder of earlier versions, say for eg: if any testnet deprecates where we used a previous version, we may need to chnage the code again right?, can we pass in the forwarder as a parameter

Reply:
I mainly thought that going forward we are going to deploy new forwarder only, but the point makes sense, will address this.

Reply:
Have added comment for now as this will need changes even in github workflow as we can take forwarder version from a label